README.md

```diff
@@ -2,6 +2,29 @@
 
 [Hack Club](https://hackclub.com/)'s [theme](https://theme.hackclub.com) implemented as CSS.
 
+## Development
```

Comment:
These aren't exactly going to help someone develop as it pulls styles from the internet, we could change how this works but for now this isn't going to help anyone

Reply:
Ah ok, I realized that something was off when I was experimenting with theme.css.

How does one dev on the site?